uaclient/security.py

```diff
@@ -572,6 +574,23 @@ def fix_security_issue_id(
                     issue_id=issue_id
                 )
             raise exceptions.UserFacingError(msg)
+
+        # Check livepatch status for CVE in fixes
```

Comment:
We can do this check before we even look up the CVE in the API. I suggest moving this block of code up to line 567.